projects/cdk/package.json

```diff
@@ -15,15 +15,13 @@
     "repository": "https://github.com/tinkoff/taiga-ui",
     "license": "Apache-2.0",
     "dependencies": {
+        "@angular-devkit/schematics": "^9.1.12",
         "@ng-web-apis/common": "^1.12.1",
         "@ng-web-apis/mutation-observer": "^1.1.0",
         "@ng-web-apis/resize-observer": "^1.0.3",
         "@tinkoff/ng-event-plugins": "^2.5.1",
         "@tinkoff/ng-polymorpheus": "^3.1.12",
-        "@types/resize-observer-browser": "^0.1.7"
-    },
-    "devDependencies": {
-        "@angular-devkit/schematics": "^9.1.12",
+        "@types/resize-observer-browser": "^0.1.7",
         "ng-morph": "^2.0.0"
```

Comment:
may be we should move them to `peerDependencies` ?

It will make our users to install `@angular-devkit/schematics` & `ng-morph` as devDependency in their projects?
Does it works in this way ?

Reply:
`may be we should move them to peerDependencies ?` 
I think we will have problems with `npm ci --legacy-peer-deps`

Reply:
yes, unfortunatelly it won't work with npm < 7 or `--legacy-peer-deps` flag